index.html

```diff
@@ -1,26 +1,36 @@
-                <div class="calculator-form__action-bar action-bar"> 
-                   <input type="button" class="action-bar__button action-bar__button-sum" id="calc-sam" onclick="calcSum();" value="Sum">
-                   <input type="button" class="action-bar__button action-bar__button-sub" id="calc-sub" onclick="calcSub();" value="Sub">
-                   <input type="button" class="action-bar__button action-bar__button-mul" id="calc-mul" onclick="calcMul();" value="Mul">
-                   <input type="button" class="action-bar__button action-bar__button-div" id="calc-div" onclick="calcDiv();" value="Div">
-                </div>
-            </form>
-            <div class="calculator-result" id="result" ></div>
+            <div class="calculator-form action-bar">
+                <input type="text" class="calculator-form__item action-bar__output" id="calc-output"  disabled>
+                <input type="button" class="calculator-form__item action-bar__button-ce" id="CE" value="CE">
+                <input type="button" class="calculator-form__item number action-bar__button-seven" id="seven" value="7">
+                <input type="button" class="calculator-form__item number action-bar__button-eight" id="eight" value="8">
+                <input type="button" class="calculator-form__item number action-bar__button-nine" id="nine" value="9">
+                <input type="button" class="calculator-form__item operators action-bar__button-div" id="div" value="&divide">
+                <input type="button" class="calculator-form__item number action-bar__button-four" id="four" value="4">
+                <input type="button" class="calculator-form__item number action-bar__button-five" id="five" value="5">
+                <input type="button" class="calculator-form__item number action-bar__button-six" id="six" value="6">
+                <input type="button" class="calculator-form__item operators action-bar__button-mul" id="mul" value="&times">
+                <input type="button" class="calculator-form__item number action-bar__button-one" id="one" value="1">
+                <input type="button" class="calculator-form__item number action-bar__button-two" id="two" value="2">
+                <input type="button" class="calculator-form__item number action-bar__button-three" id="three" value="3">
+                <input type="button" class="calculator-form__item operators action-bar__button-sub" id="sub" value="-">
+                <input type="button" class="calculator-form__item number action-bar__button-zero" id="zero" value="0">
+                <input type="button" class="calculator-form__item operators action-bar__button-sum" id="sum" value="+">
+                <input type="button" class="calculator-form__item action-bar__button-result" id="result" value="=">
+            </div>
+            <div class="calculator-bottom">
+                <a class="calculator-bottom__href" href="https://github.com/NikitaOstrovsky636">GitHub</a>
```

Comment:
Link to GitHub should be opened in a new tab (it was marked in requirements in the Trello ticket).

Reply:
Actually `calculator-bottom__href` is not the best name for it. `href` is an implementation detail. What if we decide to change `a` to `button`? Then `calculator-bottom__href`` will be an incorrect name. 

Please, rename it to `calculator-bottom__github`. That name describes the semantic of the element (link to GitHub), but not the implementation (tag `<a />`).